docs/requirements-docs.txt

```diff
@@ -12,4 +12,5 @@ torch
 py-cpuinfo
 transformers
 mistral_common >= 1.3.4
-openai # Required by docs/source/serving/openai_compatible_server.md's vllm.entrypoints.openai.cli_args
\ No newline at end of file
+openai # Required by docs/source/serving/openai_compatible_server.md's vllm.entrypoints.openai.cli_args
+partial-json-parser # Required by docs/source/serving/openai_compatible_server.md's vllm.entrypoints.openai.cli_args
```

Comment:
Are you sure these should be in the `requirements-docs.txt`? They are already in `requirements-common.txt`

Reply:
I'm sure, without this the documents can not be compiled in the ci pipline.
as the `doc/README.md` says, it's only need to install the requirements in `requirements-docs.txt`.